.github/workflows/main.yml

```diff
@@ -205,7 +205,8 @@ jobs:
           path: ${{ github.workspace }}
       # As long as there are some space issues with the CI runners.
       - name: Free Disk Space
-        uses: jlumbroso/free-disk-space@0add001504c51b93b35ac8e81d252bdd47e4ef07
+        continue-on-error: true
```

Comment:
I thin we have started running into disk-space issues fo the non-application server integration tests aswell. Could you copy this step over there, too?